libbeat/processors/dissect/dissect_test.go

```diff
@@ -66,22 +66,14 @@ func TestDissectConversion(t *testing.T) {
 			Fail: false,
 		},
 		{
-			Name: "Missing data type, drop fields",
-			Tok:  "id=%{id|} status=%{status|} msg=\"%{message}\"",
-			Msg:  "id=1857 status=404 msg=\"File not found\"}",
+			Name: "Greedy padding skip test ->",
+			Tok:  "id=%{id|integer->} padding_removed=%{padding_removed|boolean->} length=%{length|long->} msg=\"%{message}\"",
```

Comment:
If I understand correctly the proposal in https://github.com/elastic/dissect-specification/issues/10#issuecomment-616464555, the type should be after the key and its suffixes, so this should be like this:
```suggestion
			Tok:  "id=%{id->|integer} padding_removed=%{padding_removed->|boolean} length=%{length->|long} msg=\"%{message}\"",
```

Reply:
Moved data type after the keys and suffixes in regular expression and tests.